src/Microsoft.ML.Tokenizers/Model/Tiktoken.cs

```diff
@@ -495,6 +581,16 @@ static void ArrayPoolGrow(ref Span<byte> utf8Bytes, ref byte[]? arrayPoolArray,
         /// </summary>
         public override IReadOnlyDictionary<string, int> GetVocab() => _vocab;
 
+        /// <summary>
+        /// Gets the dictionary mapping token utf-8 bytes to Ids.
+        /// </summary>
+        public IReadOnlyDictionary<ReadOnlyMemory<byte>, int> Encoder => _encoder;
+
+        /// <summary>
+        /// Gets the dictionary mapping Ids to token utf-8 bytes.
+        /// </summary>
+        public IReadOnlyDictionary<int, byte[]>? Decoder => _decoder;
```

Comment:
Should we make the Decoder be `IReadOnlyDictionary<int, ReadOnlyMemory<byte>>` both so that it parallels Encoder and so that it's immutable without unsafe code?

Reply:
Done!